x-pack/plugins/observability_solution/infra/public/components/asset_details/translations.ts

```diff
@@ -10,3 +10,24 @@ import { i18n } from '@kbn/i18n';
 export const NOT_AVAILABLE_LABEL = i18n.translate('xpack.infra.assetDetails.notApplicableLabel', {
   defaultMessage: 'N/A',
 });
+
+export const HOST_METRIC_GROUP_TITLES: Record<string, string> = {
```

Comment:
Nit: We can make the type stricter - something like: 

```
type HostMetricGroup = 'kubernetes' | Exclude<HostMetricTypes, 'kpi'>;

export const HOST_METRIC_GROUP_TITLES: Record<HostMetricGroup, string> = { ...
```

Reply:
Good idea. we can let ts infer the type in this case.